android/src/main/java/software/solid/fluttervlcplayer/FlutterVideoView.java

```diff
@@ -77,19 +100,33 @@ public void onCancel(Object o) {
         textureView.setSurfaceTexture(textureEntry.surfaceTexture());
         textureView.setSurfaceTextureListener(new TextureView.SurfaceTextureListener() {
 
-            boolean wasPaused = false;
+            boolean wasPlaying = false;
+
+            private final Runnable mRunnable = new Runnable() {
+                @Override
+                public void run() {
+                    if (vout == null) return;
+                    vout.setVideoSurface(new Surface(textureView.getSurfaceTexture()), null);
+//                    vout.setWindowSize(textureView.getWidth(), textureView.getHeight());
+//                    ViewGroup.LayoutParams lp = (ViewGroup.LayoutParams) textureView.getLayoutParams();
+//                    lp.width = 300;
+//                    lp.height = 300;
+//                    textureView.setLayoutParams(lp);
+//                    textureView.invalidate();
```

Comment:
Do we need this commented out part? It looks like we need to remove all of that unless it's some additional information (if so we probably need to mark it clear) - what do you think?